service-commands/scripts/bake-dev-image.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+# bakes from latest master on gcp by default
+
+set -x
```

Comment:
`set -ex`?

Either way, <3 `set -x`!

Reply:
set `-x` is for verbose and `-e` is for exit/early/(on)error? (I don't actually know what the `e` really means! 😂 )

I definitely like having verbose so that way i can track down where I'm at if I'm debugging a wall of noise. So +1 on what's there with `set -x`.

I was suggesting turning on `set -e` as well so that the bake exits early, but it seems like your `trap` call acts like `set -e` with the additional benefit of adding a log message, which is FAR nicer!

No changes here suggested anymore :)